### PR TITLE
[DRAFT] Optimize bias

### PR DESCRIPTION
**Pull request by Aeren1564**
.

### CODE REVIEW

**Comment by Aeren1564** [comment]
== training parameter ==
- learning_rate   = 0.001
- batch_size      = 32
- loss_info       = {loss = mean squared error, reduction = sum over batch size}
- optimizer       = adam
========================
Epoch 1/5 - time: 4430.210ms/step - loss: [0] 30090.0898
Epoch 2/5 - time: 4607.027ms/step - loss: [0] 30090.0840
Epoch 3/5 - time: 4669.991ms/step - loss: [0] 30090.0840
Epoch 4/5 - time: 4705.184ms/step - loss: [0] 30090.0840
Epoch 5/5 - time: 4659.798ms/step - loss: [0] 30090.0840
===================================
MODEL_LOAD   takes 5.4760 ms
PREPARE      takes 786.9770 ms
EXECUTE      takes 207755.0330 ms
- Epoch 1      takes 39871.8920 ms
- Epoch 2      takes 41463.2420 ms
- Epoch 3      takes 42029.9200 ms
- Epoch 4      takes 42346.6520 ms
- Epoch 5      takes 41938.1790 ms
===================================



== training parameter ==
- learning_rate   = 0.001
- batch_size      = 32
- loss_info       = {loss = mean squared error, reduction = sum over batch size}
- optimizer       = adam
========================
Epoch 1/5 - time: 2800.052ms/step - loss: [0] 30090.0977
Epoch 2/5 - time: 2580.089ms/step - loss: [0] 30090.0938
Epoch 3/5 - time: 2654.771ms/step - loss: [0] 30090.0938
Epoch 4/5 - time: 2647.186ms/step - loss: [0] 30090.0938
Epoch 5/5 - time: 2648.930ms/step - loss: [0] 30090.0938
===================================
MODEL_LOAD   takes 5.3360 ms
PREPARE      takes 775.7780 ms
EXECUTE      takes 120092.1480 ms
- Epoch 1      takes 25200.4690 ms
- Epoch 2      takes 23220.8040 ms
- Epoch 3      takes 23892.9350 ms
- Epoch 4      takes 23824.6720 ms
- Epoch 5      takes 23840.3740 ms
===================================
